core/admin/mailu/api/v1/user.py

```diff
@@ -203,6 +208,8 @@ def patch(self, email):
             user_found.global_admin = data['global_admin']
         if 'enabled' in data:
             user_found.enabled = data['enabled']
+        if 'change_pw_next_login' in data:
+            user_new.change_pw_next_login = data['change_pw_next_login']
```

Comment:
The wrong variable name is used. user_new should be user_found.

```suggestion
            user_found.change_pw_next_login = data['change_pw_next_login']
```